docs/release-notes/index.md

```diff
@@ -66,6 +66,46 @@ or anything else, please submit a [GitHub issue][issues].
+| `<rh-accordion>`: improved accessibility | {{i()}} | Removed arrow-key keyboard navigation in favor of tab navigation through accordion sets. |
+| `<rh-badge>`: added new `state` colors | {{i()}} | Added `danger`, `warning`, `caution`, `neutral`, and `info`. Deprecated `critical` (aliasing `danger`), `important` (aliasing `caution`), `moderate` (aliasing `warning`), and `note` (aliasing `info`). |
+| `<rh-button>`: added `icon-set` attribute | {{i()}} | Added `icon-set="..."` attribute, which corresponds to `<rh-icon set="...">`. |
+| `<rh-card>`: added header background theming API | {{i()}} | Using `--rh-card-header-background-on-light` and `--rh-card-header-background-on-dark` CSS custom props allows for theming the card header's background. |
+| `<rh-code-block>`: added syntax highlighting | {{i()}} | Code blocks now have optional Red Hat color-themed syntax highlighting via client side or server side (prerendered prismjs code-blocks). |
+| `<rh-table>`: improved responsive layout API | {{i()}} | Added auto-generated table cell headings for responsive layout on small screens. |
+| `<rh-tag>`: added new tag colors | {{i()}} | Added `red-orange`, `yellow`, and `teal` colors. Deprecated `cyan`, aliasing it to `teal`. |
+| `<rh-tag>`: added `destaurated` variant | {{i()}} | Added `variant="desaturated"` to reduce visual prominence or to better fit a specific theme or visual style. |
+| `<rh-tag>`: added `size` attribute | {{i()}} | Added `size="compact"` for areas where space is limited. |
+| `<rh-tag>`: added optional `href` attribute | {{i()}} | Adding `href` attribute to `<rh-tag>` removes the need for slotting an anchor element (`<a>`) on linked tags. |
+| `<rh-tile>`: added `private` and `external` link variants | {{i()}} |  Using `link="private"` or `link="external"` indicates whether the link is private or external and changes the tile icon from an arrow to a padlock or external link icons respectively. |
+| `<rh-skip-link>`: added optional `href` attribute | {{i()}} | Adding `href` attribute to `<rh-skip-link>` removes the need for slotting an anchor element (`<a>`). |
+| `<rh-audio-player>`: added mini playback control | {{p()}} | Mini layout now has playback control. |
+| `<rh-navigation-secondary>`: improved accessibility | {{p()}} | Removed arrow-key keyboard navigation in favor of tab navigation through navigation items and added `accessible-label` attribute to explicitly label landmark. |
+| `<rh-subnav>`: improved accessibility | {{p()}} | Removed arrow-key keyboard navigation in favor of tab navigation through navigation items and added `accessible-label` attribute to explicitly label landmark. |
+| `<rh-table>`: added container query support | {{p()}} | Tables now adjust to the size of their containing element, not just viewport size. |
```

Comment:
```Suggestion
| `<rh-table>`: added container query support | {{p()}} | Tables now adjust to the size of their containing element, not the viewport size. |
```

Reply:
Media queries were removed completely, the "added support" here was more or less a replacement of @media for @container